go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/googleapis/gapic-generator-go
 
 require (
+	github.com/ghodss/yaml v1.0.0
 	github.com/golang/protobuf v1.5.2
 	github.com/google/go-cmp v0.5.7
 	github.com/jhump/protoreflect v1.12.0
 	gitlab.com/golang-commonmark/markdown v0.0.0-20211110145824-bf3e522c626a
 	google.golang.org/genproto v0.0.0-20220414192740-2d67ff6cf2b4
 	google.golang.org/protobuf v1.28.0
-	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Out of curiosity, any reason for this switch? This is what I would consider the more popular lib in the ecosystem

Reply:
`gopkg.in/yaml.v2` doesn't have a Yaml-to-JSON converter, which is what I need to correctly parse the contents using the protojson decoder. There isn't a protoyaml decoder, so :)